src/Microsoft.Data.SqlClient/tests/UnitTests/Microsoft.Data.SqlClient.UnitTests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;net6.0</TargetFrameworks>
```

Comment:
Given that these are only for .NET Runtime, could we name the project accordingly?
NetCoreUnitTests?

Reply:
Sounds good. I can do that. 

`Microsoft.Data.SqlClient.NetCore.UnitTests` 
Sounds ok ?